collection/sites/twitter.yml

```diff
@@ -0,0 +1,11 @@
+name: Twitter
```

Comment:
Din't work for me with `x.com` links on a clean 3.3.10 install. Worked after changing the name attribute, f.e. to `Twitter/X.com`.

Reply:
Well, I've made a fresh phpBB installation and then installed new ext code.
But ok, I'll double check this.

Reply:
Asking cuz it works fine for me as is, which is also how it is currently set up in the latest from s9e. Also the name field should not affect the embedding.

Reply:
Okay, rechecked and it worked this time. Probably there're some Interbet connection issues on my side.